superset/datasets/api.py

```diff
@@ -195,7 +195,11 @@ class DatasetRestApi(BaseSupersetModelRestApi):
         "owners": RelatedFieldFilter("first_name", FilterRelatedOwners),
         "database": "database_name",
     }
-    search_filters = {"sql": [DatasetIsNullOrEmptyFilter]}
+    search_filters = {
+        "sql": [DatasetIsNullOrEmptyFilter],
+        "id": [DatasetCertifiedFilter],
+    }
+    search_columns = ["id", "database", "owners", "sql", "table_name"]
```

Comment:
I believe that this limits search to only these columns, is that the intention?

Reply:
I did this with my filter PR and it broke a lot of things.

Reply:
if i don't set the search columns, the new filter won't be referenced. So i'm explicitly setting it here just like the other API classes